core/src/sql/function.rs

```diff
@@ -20,7 +20,7 @@ use super::Kind;
 
 pub(crate) const TOKEN: &str = "$surrealdb::private::sql::Function";
 
-#[revisioned(revision = 1)]
+#[revisioned(revision = 2)]
```

Comment:
I don't think we need to use revision here. Just ensure that the enum variant is added at the end (before the `Add new variants here` comment).